wordpress-updatable/versions/0.1.3/schema.yaml

```diff
@@ -0,0 +1,122 @@
+x-google-marketplace:
+  schemaVersion: v2
+
+  applicationApiVersion: v1beta1
+
+  publishedVersion: '0.1.3'
+  publishedVersionMetadata:
+    releaseNote: >-
+      A new and improved release.
```

Comment:
nit: make this actually descriptive, i.e. "Use supported apiVersions" (which might be more `BugFix` than `Feature`)

Docs: https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/7e45a078237a01ee6af649815ba914646d320aa7/docs/managed-updates.md#top-level-x-google-marketplace-section